### PR TITLE
Update ITTypeSplit.kt

### PR DESCRIPTION
**Pull request by flauta**
moved vowels to right, consonants to left
reordered on letter frequency order
added schwa and reversed-E symbols

### CODE REVIEW

File: app/src/main/java/com/dessalines/thumbkey/keyboards/ITTypeSplit.kt [hunk 1]
```diff
@@ -547,7 +574,7 @@ val KB_IT_TYPESPLIT_SHIFTED = KeyboardC(
 )
 
 val KB_IT_TYPESPLIT: KeyboardDefinition = KeyboardDefinition(
-    title = "italiano type-split",
+    title = "IT Type-Split italiano",
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/keyboards/ITTypeSplit.kt:577]: Change back to the old title.

Comment (flauta) [reply app/src/main/java/com/dessalines/thumbkey/keyboards/ITTypeSplit.kt]: sorry @dessalines, I have updated the line changing it back but I don't know what I should do now

Comment (dessalines) [reply app/src/main/java/com/dessalines/thumbkey/keyboards/ITTypeSplit.kt]: Its no problem, I'll clean this and the other PRs up when I do the next release, just keep it open for now.

**Comment by dessalines** [comment]
Even before my merge from main, this was riddled with typos and errors. Please do a better job next time.

**Review by dessalines — APPROVED** [review]

**Comment by flauta** [comment]
Thanks @dessalines for fixing the PR, and to accept a low-quality work as my work is. 
I'm not a professional programmer and I can't compile on my own, so if you can suggest me a way to check my code before committing (other than reading many times the source as I do), maybe I'd be able to reduce or avoid typos and errors.

**Pull request merged**